copy_proto_test.go

```diff
@@ -191,6 +191,37 @@ func TestStructToStruct_MaskInverse(t *testing.T) {
 	assert.Equal(t, userSrc, userDst)
 }
 
+func TestStructToStruct_MaskInverseFromMask(t *testing.T) {
+	userSrc := &testproto.User{
+		Id:          1,
+		Username:    "username",
+		Role:        testproto.Role_ADMIN,
+		Meta:        map[string]string{"foo": "bar"},
+		Deactivated: false,
+		Permissions: []testproto.Permission{testproto.Permission_EXECUTE},
+		Name:        &testproto.User_FemaleName{FemaleName: "Dana"},
+		Friends: []*testproto.User{
+			{Id: 2, Username: "friend1"},
+			{Id: 3, Username: "friend2"},
+		},
+	}
+	userDst := &testproto.User{}
+	// Mask to MaskInverse
+	mask := fieldmask_utils.MaskInverse(fieldmask_utils.Mask{"Id": fieldmask_utils.Mask{}, "Friends": fieldmask_utils.Mask{"Username": fieldmask_utils.Mask{}}})
```

Comment:
I think the right way to do that is to create a new helper function `MaskInverseFromMask(Mask) MaskInverse` that will create a `MaskInverse` from a given `Mask` recursively (including all the submasks)

Reply:
Maybe no need to create one.
for MaskInverse: subFilter.IsEmpty() means the sub should be filtered.

Reply:
I've added some tests, PTAL https://github.com/mennanov/fieldmask-utils/commit/83cb71b9bb683e1d733faa67c292cf392f43b483

Is that what you're looking for?

Reply:
yes, nil and empty should be same strategy when convert from Mask to MakeInverse.